### DISCUSSION TITLE
Different format for PR titles created by dependabot

### DISCUSSION THREAD

**Discussion by VitaliySerov** [dependabot/dependabot-core#10995] [Q&A] [answered]
Hi

I got plenty of repos I need to manage and update by dependabot
In 99% of my repos ,the PR created by dependabot is titles like this [one](https://github.com/ONLYOFFICE-QA/onlyoffice_s3_wrapper/pull/1506):

`Bump aws-sdk-kms from 1.95.0 to 1.96.0`

But in one or two repos the title is for some reason is like this [one](https://github.com/ONLYOFFICE-QA/testing-api.onlyoffice.com/pull/2329):

`build(deps): bump aws-sdk-kms from 1.95.0 to 1.96.0`

But the interesting thing is that I've compared `dependabot.yml` file in both repos - and they are exactly the same, line by line
The `commit-message` line is not changed, so it uses the default value

Any idea why this happened?

**Comment by ReenigneArcher** [comment]
Dependabot will try to conform to conventional commits (2nd example) if it thinks you use that in your repo otherwise it will use the first example.

**Reply by VitaliySerov** [reply]
Any documentation where it describes how it determines this logic? 
I prefere my config files clean, so maybe there is a way to change it without touching `commit-message` option (or any other option)

**Reply by ReenigneArcher** [reply, marked as answer]
I'm not sure if it's documented, but I also noticed the same behavior as you did. My earlier repos before I started using conventional commits were using your first example, and newer repos were using the second option.

It will change over if you change the commit styles in your repo though, so if you want one or the other you need to use that convention outside of dependabot as well. I'm not sure how much of the history it looks at. Personally, I have a workflow run on PRs in my org to ensure they follow conventional commit guidelines.

**Reply by VitaliySerov** [reply]
Thanks for explanation
I think in my case simpler solution will be hard-coded commit format in dependabot config, not sure what I want to 'commit undefined count of commit messages' to get things in order

**Reply by tiagoporto** [reply]
@ReenigneArcher I guess I'm facing the same issue https://github.com/dependabot/dependabot-core/issues/11806.
Do you know how can I force dependabot uses the default format?

**Reply by ReenigneArcher** [reply]
@tiagoporto I'm not sure. I just know it looks at the history of your commits to decide.
